src/Sulu/Bundle/ContactBundle/Entity/Contact.php

```diff
@@ -32,6 +33,8 @@ class Contact extends ApiEntity implements ContactInterface, AuditableInterface
 {
     /**
      * @var int
+     * @Expose
+     * @Groups({"frontend"})
```

Comment:
Also didn't find a better solution for now, might improve if we handle the JMS Serializer configuration the same way as the doctrine stuff in the PersistenceBundle.